packages/api/app/testAuth.js

```diff
@@ -65,7 +69,7 @@ async function refreshEndpoint(req, res) {
  * @returns {Promise<Object>} a promise of an express response
  */
 async function deleteTokenEndpoint(req, res) {
-  token = '';
+  accessToken = '';
```

Comment:
clearly nothing on inspection.  Looking at master, it appears that it really didn't do anything except set the "authentication" comparer to and empty string so that ensure authorized would fail on the next attempt.  This looks like cruft from previous, possibly bad refactors.    I'll see if I can make sense of it.

Reply:
handled. 9c39258e2